docs/src/_concepts/project.md

```diff
@@ -251,33 +251,57 @@ Commands can specify a `container_spec` for containerized execution. To execute
         ports:
           "8080": "8080/tcp"
 ```
+### Jobs
+
+Your project's predefined pipelines, typically created using [`meltano job`](/reference/command-line-interface#job), are defined under the `jobs` property.
+
+A job definition must have a `name` and one or more `tasks`:
+
+```yaml
+jobs:
+- name: tap-foo-to-target-bar-dbt
+  tasks:
+    - tap-foo target-bar dbt:run
+- name: tap-foo-to-targets-bar-and-baz
+  tasks:
+    - tap-foo target-bar
+    - tap-foo target-baz
+```
+
 
 ### Schedules
 
 Your project's pipeline schedules,
 typically [created](/guide/orchestration#create-a-schedule)
 using [`meltano schedule`](/reference/command-line-interface#schedule),
- are defined under the `schedules` property.
+are defined under the `schedules` property.
 
-A schedule definition must have a `name`, `extractor`, `loader`, `transform` and `interval`:
+A schedule definition must have a `name`, `job` and `interval`:
```

Comment:
I'm not sure if this intentional because we're steering people away from elt schedules, but a job is not required. A schedule must have a `name`, `interval`, and either `job` or "elt" definition. 

Maybe you could say something like:

"A scheduled _job_ must have..."

Reply:
Yeah this was intentional but agreed that specifying that it's for jobs is better.

Reply:
👍 cool makes sense in that case.